Gruntfile.js

```diff
@@ -23,22 +22,10 @@ module.exports = function(grunt) {
       production: 'publish',
       bamboo: 'bamboo',
 
-      // webrtc-adapter submodule
-      webrtcAdapterSubmodule: 'third_party/adapter/',
-      googleAdapterPath: '<%= webrtcAdapterSubmodule %>/out/adapter.js',
-
       pluginInfoRoot: grunt.option('pluginInfoRoot') || '<%= source %>',
       pluginInfoFile: grunt.option('pluginInfoFile') || 'pluginInfo.js',
 
-      githash: {
-        submodule: {
-          options: {
-            dir: '<%= webrtcAdapterSubmodule %>'
-          }
-        },
-      },
-
-      version: '<%= pkg.version %>-<%= githash.submodule.short %>',
+      version: '<%= pkg.version %>',
```

Comment:
I'm fine with this limitation. We can always revert in the future if we realise this is completely blocking for us.
It does sound safer to only use released version anyway. It means that it passed Google's safety checks.

@NTARelix , could you change your PR so that it uses a specific version rather than a compatible one ? (2.0.3 instead of ^2.0.3)